.gitignore

```diff
@@ -7,3 +7,4 @@ BUILDROOT
 RPMS
 SOURCES
 SRPMS
+.idea
```

Comment:
Please do not add this line to `.gitignore` as part of this PR. Such change is not related to this PR.